src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,437 @@
+| InjectedMassUnit              | REQUIRED | [string][] | Unit format of the mass of compound injected (for example, "ug" or "umol"). **Note this is not required for an FDG acquisition, since it is not available, and can be set to "-1"**.                                                                                       |
+| SpecificRadioactivity         | REQUIRED | [number][] | Specific activity of compound injected. **Note this is not required for an FDG acquisition, since it is not available, and can be set to -1**.                                                                                                                             |
+| SpecificRadioactivityUnit     | REQUIRED | [string][] | Unit format of specified specific radioactivity (for example, "Bq/g"). **Note this is not required for an FDG acquisition, since it is not available, and can be set to "-1"**.                                                                                            |
+| ModeOfAdministration          | REQUIRED | [string][] | Mode of administration of the injection (for example, "bolus" or "infusion").                                                                                                                                                                                              |
+| InjectedMassPerWeight         | RECOMMENDED | [number][] | Injected mass per kilogram bodyweight.                                                                                                                                                                                                                                  |
+| InjectedMassPerWeightUnit     | RECOMMENDED | [string][] | Unit format of the injected mass per kilogram bodyweight (for example, "ug/kg").                                                                                                                                                                                        |
+| SpecificRadioactivityMeasTime | RECOMMENDED | [string][] | Time to which specific radioactivity measurement above applies in the default unit "hh:mm:ss".                                                                                                                                                                          |
+| MolarActivity                 | RECOMMENDED | [number][] | Molar activity of compound injected. Corresponds to DICOM Tag (0018,1077) Radiopharmaceutical Specific Activity.                                                                                                                                                        |
+| MolarActivityUnit             | RECOMMENDED | [string][] | Unit of the specified molar radioactivity (for example, "Bq/g").                                                                                                                                                                                                        |
```

Comment:
```suggestion
| MolarActivityUnit             | RECOMMENDED | [string][] | Unit of the specified molar radioactivity (for example, "GBq/umol").                                                                                                                                                                                                        |
```

Bq/g is specific activity. I've changed it to GBq/umol . This matches the example JSON data later down the page. However, from the chemistry consensus paper, they state that GBq/umol or Bq/mol can both be used (and they're both equivalent: same number). Would Bq/mol not be preferable (in both the example and the table) as it involves no multiple prefixes?

Reply:
First, thanks a lot for your comments, @mathesong. You are absolutely right. This should be Bq/mol.

Reply:
Actually, thinking this through again, wouldn't GBq/umol be a factor of 1000 greater than Bq/mol..?

The chemistry consensus paper puts it like this as if they are equivalent:

![image](https://user-images.githubusercontent.com/7190884/104717561-c9738980-5729-11eb-87af-703404dbe1f1.png)

and the guidelines paper puts it like this:

![image](https://user-images.githubusercontent.com/7190884/104717872-4a328580-572a-11eb-8d8e-005b2ffebae1.png)

I read this initially as their being eqivalent, but, maybe I'm being stupid, but isn't this 10^9 Bq / 10^-6 mol .  Or I guess it's probably that it's just fine to report in either Bq/mol or GBq/umol, even though they're different values.  But in this case, probably best to stick with GBq/umol (though not Bq/g as it was before) in the example to match the first suggestion in both papers.

Reply:
I am confused. Would it not be 

```
    GBq       10^6 Bq                 Bq
1 * ---- =  ------------ = 10^12  *  ----
    umol      10^-6 mol               mol
```
?

Reply:
Haha - now we're all confused :).  Worse, I think we're all wrong actually.

G = giga = 10^9, --> 1 Bq = 10^-9 GBq     (10^6 would be mega, M)
u = micro = 10^-6, --> 1 mol = 10^6 umol

so:

![image](https://user-images.githubusercontent.com/7190884/104729067-29712c80-5738-11eb-8caf-ef02699f02a2.png)

Sorry for all the confusion!

Reply:
Dear @mathesong, I think we should stick with Bq/mol and this is as we saw above not equivalent to Gbq/umol. But then people are not confused if we just force everyone to convert it to Bq/mol.

Reply:
I just came back from a little break, and I read through the above again, and I hang my head in shame once more to have to report that I seem to have got it wrong once again.  I even have an (old, and mostly unused) undergrad degree in applied chemistry to my name, so this is doubly embarrassing 😬😂 - but I'll blame it on its still being January. @robertoostenveld was closest though :)

Here's what it should be:

![image](https://user-images.githubusercontent.com/7190884/105489705-f7ba1180-5cb3-11eb-885e-1c4e1af28097.png)

As such, I think I now understand why both were presented: Bq/mol is the basic SI unit, but it's a very, very large number and it's probably highly prone to error when having to count out all those zeros.  I think forcing everyone to use GBq/umol is probably preferable, despite the multipliers.

Reply:
I agree with Granville. Also on the math
![Blank diagram (1)](https://user-images.githubusercontent.com/12412821/105519732-13381300-5cda-11eb-86e4-8a3c1eb236fb.png)